docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+   | global_cycle           | Updates the GFS surface conditions using external snow and sea ice analyses     |
+   +------------------------+---------------------------------------------------------------------------------+
+   | inland                 | Creates an inland land mask by determining in-land (i.e. non-coastal) points    |
+   |                        | and assigning a value of 1. Default value is 0.                                 |
+   +------------------------+---------------------------------------------------------------------------------+
+   | orog_gsl               | Ceates orographic statistics fields required for the orographic drag suite      |
+   |                        | developed by NOAA's Global Systems Laboratory (GSL)                             |
+   +------------------------+---------------------------------------------------------------------------------+
+   | fregrid                | Remaps data from the input mosaic grid to the output mosaic grid                |
+   +------------------------+---------------------------------------------------------------------------------+
+   | lakefrac               | Calculates the ratio of the lake area to the grid cell area at each atmospheric |
+   |                        | grid point.                                                                     |
+   +------------------------+---------------------------------------------------------------------------------+
+
+.. _Data:
+
+Download and Stage the Data
+============================
+
+The SRW App requires input files to run. These include static datasets, initial and boundary conditions files, and model configuration files. On Level 1 and 2 systems, the data required to run SRW App tests are already available. For Level 3 and 4 systems, the data must be added. Detailed instructions on how to add the data can be found in the :numref:`Section %s Downloading and Staging Input Data <DownloadingStagingInput>`. :numref:`Sections %s <Input>` and :numref:`%s <OutputFiles>` contain useful background information on the input and output files used in the SRW App. 
+
+.. _GridSpecificConfig:
+
+Grid Configuration
+=======================
+
+The SRW App officially supports three different predefined grids as shown in :numref:`Table %s <PredefinedGrids>`. The "out-of-the-box" SRW App case uses the ``RRFS_CONUS_25km`` predefined grid option. More information on the predefined and user-generated grid options can be found in :numref:`Chapter %s <LAMGrids>` for those who are curious. Users who plan to utilize one of the three pre-defined domain (grid) options may continue to :numref:`Step %s <GenerateForecast>`. Users who plan to create a new domain should refer to :numref:`Chapter %s <LAMGrids>` for details on how to do so. At a minimum, these users will need to add the new grid name to the ``valid_param_vals`` script and add the corresponding grid-specific parameters in the ``set_predef_grid_params`` script. 
```

Comment:
The difficulty with "domain" is that it's both a more general and a more technical term. When I hear "domain," I think of domain (X) vs range (Y) as taught in high school/college math classes. It's not immediately clear from the term that we're talking about _a region/area of the globe_ as input to a model. For people with domain-specific knowledge (pun intended ;) ), it's probably pretty clear that domain refers to _spacial_ domain. I like "grid" because it has the benefit of imagery. People can picture a grid; the globe is already divided by latitude/longitude, which is grid-like, so it's clearer in my mind what we're talking about. That said, I think that pretty much anyone considering using the UFS applications probably has a good deal of subject-specific knowledge/context, so maybe writing for a broader audience is less important here. My two cents. ;)